.github/workflows/integration.yaml

```diff
@@ -28,6 +28,8 @@ jobs:
       - uses: actions/checkout@v4
         with:
           persist-credentials: false
+      - name: Setup Poetry
+        uses: snok/oinistall-poetry@v1
```

Comment:
I think we can get away with not using this action:
```
sudo apt-get install python3-pip python3-venv -y --no-install-recommends
python3 -m pip install --user pipx
python3 -m pipx ensurepath
echo "${HOME}/.local/bin" >> "${GITHUB_PATH}"

pipx install poetry
```

Reply:
Fixed